### PR TITLE
Plat 114422 - Add fields to destination and destination-segment schema

### PR DESCRIPTION
**Pull request by tumulurik**
[Please link to the issue #…]
PLAT-114422 - https://jira.corp.adobe.com/browse/PLAT-114422

### CODE REVIEW

**Comment by pareshshenoy** [comment]
👍

File: schemas/destinations/destination.schema.json [hunk 1]
```diff
@@ -83,6 +83,16 @@
           "description": "Unique identifier to connect and deliver segments to the destination",
           "meta:titleId": "destination##xdm:connectionSpecID##title##40981",
           "meta:descriptionId": "destination##xdm:connectionSpecID##description##44091"
+        },
+        "xdm:destinationCategory": {
```
Comment (mjasrotia) [inline schemas/destinations/destination.schema.json:87]: It will help to have enums defined for this field is you have a list

Comment (tumulurik) [reply schemas/destinations/destination.schema.json:87]: New categories get added as new destinations are worked on.

Comment (mjasrotia) [reply schemas/destinations/destination.schema.json:87]: Sounds Good

**Review by mjasrotia — APPROVED** [review]

File: schemas/destinations/destination.schema.json (hunk 1 above)
Comment (tumulurik) [reply schemas/destinations/destination.schema.json:87]: Will update with the enum values for this field as well as the destinationFrequency.

Comment (tumulurik) [reply schemas/destinations/destination.schema.json:87]: Pushed the enum values.

**Pull request merged**